test/test_markdown2.py

```diff
@@ -166,7 +166,9 @@ def charreprreplace(exc):
                           _display(python_toc_html), _display(toc_html),
                           _indent(diff))
             self.assertEqual(python_norm_toc_html, norm_toc_html,
-                errmsg.encode('ascii', 'charreprreplace'))
+                errmsg.encode('utf-8', 'charreprreplace'))
+            # self.assertEqual(python_norm_toc_html, norm_toc_html,
+            #     errmsg.encode('ascii', 'charreprreplace'))
```

Comment:
sure, but I'm not sure about whether the code need to be this, to CI and me, it works fine.